GHzDACs/Cleanup/dac.py

```diff
@@ -71,38 +128,39 @@
 # to raw byte strings prior to being sent to the direct ethernet server.
 
 # Time for master to delay before SRAM to ensure synchronization
-MASTER_SRAM_DELAY = 2 #us
+MASTER_SRAM_DELAY = 2  # us
 
 
 class DAC(fpga.FPGA):
-    
     MAC_PREFIX = '00:01:CA:AA:00:'
     REG_PACKET_LEN = 56
     READBACK_LEN = 70
-    
+
     TIMING_PACKET_LEN = 30
-    
+
     # Master delay before SRAM to ensure synchronization
-    MASTER_SRAM_DELAY = 2 # microseconds
+    MASTER_SRAM_DELAY = 2  # microseconds
+
+    HAS_JUMP_TABLE = False
```

Comment:
About this: this is only used in the FPGA server to check if the user called JT FPGA settings with a non-JT board selected, with the purpose of giving a more helpful error message than `class [whatever] has no attribute jt_blah_blah`. I think the "right" approach would be to implement all these functions in the base class and raise a `NotImplementedError("Board version {} does not support this_function".format(self.build_number)`. A mixin seems a bit too complicated for something like this.